requirements.txt

```diff
@@ -1,6 +1,6 @@
-google-cloud-alloydb-connector[asyncpg]==1.4.0
-google-cloud-storage>=2.18.2, <3.0.0
```

Comment:
Should this restriction go away?

Reply:
That restriction is set in the pyproject.toml